docs/cloud/account-management/billing-and-usage.mdx

```diff
@@ -1,8 +1,23 @@
 ---
-title: Billing & Usage
+title: 'Billing & Usage | Cypress Cloud'
+description: 'See how many tests you recorded in Cypress, billing plans, pricing, and when your plan renews'
+sidebar_label: Billing & Usage
 sidebar_position: 50
 ---
 
+:::info
+
+##### <Icon name="question-circle" color="#42a1c0" /> What you'll learn
+
+- How to view billing and usage information
+- How to request an Open Source Plan
+
+:::
```

Comment:
is the `What you'll learn` needed? I find these badges are pretty distracting and usually explained by the sub nav titles

Reply:
I had a long convo with Brian about the info sections and we decided to keep them. They're not consistent on every page, but I do see the value outside of the side nav. Plus not everyone see the side nav on mobile.